doc/input-output-reference/src/input-for-output.tex

```diff
@@ -2081,10 +2081,91 @@ \subsubsection{Inputs}\label{inputs-12-014}
 
 Currently, there are limited capabilities for specifying SQLite generation.~ The Simple option will include all of the predefined database tables as well as time series related data.~ Using the SimpleAndTabular choice adds database tables related to the tabular reports that are already output by EnergyPlus in other formats.~ The description for SQLite outputs is described fully in the Output Details document.
 
-And one includes it in the IDF using the following.
+\paragraph{Field: Tabular Unit Conversion}\label{field-tabular-unit-conversion}
 
+When the Option Type field is set to SimpleAndTabular so that the tabular reports are included in the SQL file, this field is active. The Tabular Unit Conversion field determines if any unit conversions will be performed for the values of the tabular reports that are included in the SQL file. This field only affects the unit conversions for the tabular reports in the SQL file and does not convert the units of the time series data in the SQL file. It also has no impact on the unit conversions for the tabular reports in the HTML or other output formats. 
```

Comment:
"tabular" appears so many times in these sentences, it's confusing. How about this:

This field determines if any unit conversions will be performed for the values in the tabular reports when the Option field is set to SimpleAndTabular. These conversions are only applied to the tabular reports in the SQL file. No conversions are applied to the time series data and predefined database tables generated by the Simple option (see list below).

Reply:
Done.